### DISCUSSION TITLE
Templating inside tap actions

### DISCUSSION THREAD

**Discussion by ricoroodenburg** [home-assistant/frontend#17886] [Dashboard]
Currently it is not possible to use templates in a tap action, for example in an URL.

For some purposes it can be very useful to use attributes of an entity inside a URL.

For example: use the attribute ‘IP address’ of a virtual machine entity, inside an URL tap action for opening a RDP connection with the RD Client app by the rdp:// URI.

**Comment by ricoroodenburg** [comment]
I already open another FR on the community. I think on Github is the best place related to dashboards.

https://community.home-assistant.io/t/template-in-tap-action/612404

**Comment by karwosts** [comment]
I think the way to do this is to just create a script, and then have the tap action run the script?

**Reply by ricoroodenburg** [reply]
Hmm I don’t know for sure. But you can’t open a another app by URI with a script isn’t it? Same for URL to a webpage.

**Reply by karwosts** [reply]
Ah right you're probably correct.
